root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4848,6 +5405,11 @@ testData.forEach(function (subtest, i) {
       }
       tested = true;
     }
+    if (subtest.expected_favicon_class) {
+      const actualFaviconClass = getFaviconClass(actualCleanUrl);
+      st.equal(actualFaviconClass, subtest.expected_favicon_class, 'Correct favicon');
```

Comment:
Should it test the input URL too? For existing unclean URLs.

Reply:
I was expecting we wouldn't necessarily care if the favicon is not shown for existing URLs that need fixing anyway :) (that said, wonder if we could / should have some sort of either report or automatic script for fixing those).